library.properties

```diff
@@ -1,7 +1,7 @@
 name=Blues Wireless Notecard
 version=1.5.1
 author=Blues Wireless
-maintainer=Blues Wireless <info@blues.io>
+maintainer=Blues <info@blues.com>
```

Comment:
ok to change the maintainer name and email? @zfields

Reply:
Yes. You could even do the author while you're at it.